lte/gateway/c/oai/tasks/sgw/sgw_handlers.c

```diff
@@ -685,145 +726,176 @@ int sgw_handle_sgi_endpoint_deleted(
   OAILOG_FUNC_RETURN(LOG_SPGW_APP, rv);
 }
 
+//------------------------------------------------------------------------------
+static void populate_sgi_end_point_update(
```

Comment:
Could you put a comment about the function? Also in the signature, it is better to put the arguments to be modified in the function call at the end for readability.